src/components/ContextMenu/ContextMenu.module.scss

```diff
@@ -0,0 +1,43 @@
+.contextMenu {
+  display: none;
+  position: absolute;
+  top: -0.45em;
+  transform: translateY(-100%);
+  white-space: nowrap;
+  user-select: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  background-color: #c4e3f7;
+  border: 1px solid #09f;
+  border-radius: 2px;
+  -webkit-filter: drop-shadow(0px 0px 0.25em rgba(44, 44, 44, 0.75));
```

Comment:
`-webkit-filter` is not needed, as all browsers we support already support `filter`. https://caniuse.com/css-filters